### DISCUSSION TITLE
Disable issues and pull requests in subtree repos

### DISCUSSION THREAD

**Discussion by andyexeter** [lhapaipai/symfony-vite-dev#20] [Ideas]
I think we should disable issues and pull requests in the https://github.com/lhapaipai/vite-plugin-symfony and https://github.com/lhapaipai/vite-bundle repositories and update the readmes of both to point people to this repository.

Having issues and pull requests open for all three repositories is confusing in my opinion and we can't accept PRs on the subtree repos anyway.

**Comment by lhapaipai** [comment]
I am aware that the monorepo is really nicer for the development experience, that the subtree split solves the problem of sending only the bare essentials to packagist.
but this can be confusing for users...

unfortunately I looked and I believe that it is not possible to deactivate PR for a repository. maybe I missed something? to have a single place for issues of course.

**Comment by lhapaipai** [comment]
If I had not deactivated the sub-repository issues it is because I feared that the developers would end up on a repository with no issue and I often find it interesting when I have a problem to look at a solution in the issues history

**Comment by andyexeter** [comment]
Ah yes, it seems you can't disable pull requests. It looks like Symfony have an automated GitHub action which auto-closes pull request on subtree split repositories e.g: https://github.com/symfony/console/pull/85#issuecomment-1798055304

I agree it's interesting to look at past issues, however I believe the current set up is quite confusing as you'd need to look at issues in three different repositories to try to find a solution, rather than a single location.

I wonder if there is a way to carry over the past issues from the subtree split repos to the main repo?

**Comment by andyexeter** [comment]
For PRs, we could add a [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) to both of the subtree split repos encouraging users to submit the PR against the main repo instead

**Comment by lhapaipai** [comment]
yes I think it is really necessary :laughing: ...

**Comment by lhapaipai** [comment]
ultimately I did not deactivate the issues because this leaves us the possibility of referring to their content. on the other hand I added explicit templates for the issues and pr
